aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1837,6 +1837,16 @@ impl AptosVM {
         let (prologue_change_set, mut user_session) = unwrap_or_discard!(prologue_session
             .into_user_session(self, &txn_data, resolver, change_set_configs, code_storage,));
 
+        // Disallow write in prologue session for now.
+        if !self.features().is_allow_write_in_prologue_session_enabled()
```

Comment:
So the feature is off by default?

Reply:
i don't think there is a way to let the feature be on by default .